drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1616,6 +1653,32 @@ function DragAndDropBlock(runtime, element, configuration) {
                 $container.addClass('dragging');
                 $root.find('.target .zone').first().focus();
             }
+
+            // Enable sliding in accessibility mode
+            if (DragAndDropBlock.isItemSizingFixed()) {
+                var $item = $(this);
+                var itemNumber = $item[0].attributes['data-value'].value;
+                var maxItems = $container.find('.option[draggable=true]').length;
+
+                if (isBrowsingForwardsInAccessibility(evt)) {
+                    if (itemNumber>1 && itemNumber%8 === 7) {
```

Comment:
nit: spacing around `>` and `%`
```suggestion
                    if (itemNumber > 1 && (itemNumber % 8) === 7) {
```